### DISCUSSION TITLE
How can I tell the importer app that it should be using https?

### DISCUSSION THREAD

**Discussion by alexklibisz** [firefly-iii/firefly-iii#9128] [Firefly III] [answered]
I'm running the importer behind nginx, which is handling TLS.

The importer home page loads, but it shows up without stylesheets and the form is reported as insecure:

<img width="923" alt="image" src="https://github.com/user-attachments/assets/224fec0c-a975-41ab-8532-86df6dfa6e30">

<img width="471" alt="image" src="https://github.com/user-attachments/assets/b9ba05a4-537b-42f5-9f14-fcd82a26d1db">

When I look in the dev tools, it seems that the frontend is attempting to request resources and post a form to the http URL:

<img width="1081" alt="image" src="https://github.com/user-attachments/assets/fc01629f-655b-42f5-b022-cca6171735c6">

So my question is, **is there any way to tell the importer that it should be using an https URL?

If not, how exactly should we be hosting this app with TLS/https?

**Comment by JC5** [comment, marked as answer]
https://docs.firefly-iii.org/references/faq/install/#how-do-i-set-tls-in-firefly-iii-or-the-data-importer
